book/src/asm/instructions.md

```diff
@@ -0,0 +1,30 @@
+# Instructions
+
+Instructions are declared as part of a powdr asm machine. Their inputs and outputs are [assignment registers](./registers.md) as well as labels. Once defined, they can be called by any function of this machine.
+
+# Local instructions
+
+A local instruction is the simplest type of instruction. It is called local because its behavior is defined using constraints over columns of the machine it is defined in.
```

Comment:
```suggestion
A local instruction is the simplest type of instruction. It is called local because its behavior is defined using constraints over registers and columns of the machine it is defined in.
```